jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/XGBoost.scala

```diff
@@ -351,10 +419,24 @@ object XGBoost extends Serializable {
     }
   }
 
+  /**
+   * Check if the Rabit tracker returned successfully and create a model from the boosters. If
+   * tracker exited unsuccessfully, clean up and throw an error to indicate training failed.
+   *
+   * @note Copies of the finished model reside in each partition of the `distributedBoosters`.
+   *       Any of them can be used to create the model. Here, just choose the first partition.
+   *
+   * @param trackerReturnVal Exit code of the Rabit tracker.
```

Comment:
@CodingCat In this case, I think it's perfectly reasonable to remove the parameter annotations, since they are pretty self-explanatory. I think the description, especially the note, is helpful.

Reply:
please remove the comments block and note part to the line right above `.first()`